packages/legacy-client/src/client/objectSets/OsdkObjectSet.ts

```diff
@@ -108,8 +109,16 @@ export function createOsdkObjectSet<
     },
     select<T extends keyof SelectableProperties<OsdkLegacyObjectFrom<O, K>>>(
       properties: T[],
-    ): FilteredPropertiesTerminalOperations<OsdkLegacyObjectFrom<O, K>, T[]> {
-      throw new Error("not implemented");
+    ): FilteredPropertiesTerminalOperationsWithGet<
+      OsdkLegacyObjectFrom<O, K>,
+      T[]
+    > {
+      return createFilteredPropertiesObjectSetWithGetTerminalOperationsStep(
+        client,
+        apiName,
+        objectSetDefinition,
+        properties,
+      );
```

Comment:
This returned object has a `get` method on it, which is guarded against by the interfaces. I can rework the component parts to remove it if you think that's important.